linchpin/provision/roles/libvirt/files/schema.json

```diff
@@ -15,6 +15,7 @@
                     "vcpus": { "type": "integer", "required": true },
                     "cpu_mode": { "type": "string", "required": false },
                     "memory": { "type": "integer", "required": true },
+                    "static_ip": {"type": "boolean","required": false },
```

Comment:
I think this should be a network option: VM might be connected on many networks, but only for some its important to have a static dhcp